### PR TITLE
Update 14-backup-restore.md

### PR DESCRIPTION
**Pull request by Jesperbelt**
Quick-fix; How to install TrueTool

Signed-off-by: Jesperbelt <70942135+Jesperbelt@users.noreply.github.com>

**Description**
Just a small doc change of https://truecharts.org/docs/manual/SCALE%20Apps/Quick-Start%20Guides/backup-restore
⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
Its original readme docs from truetool.

**📃 Notes:**
I did add extra commentary underneath How to install.

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [x] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [x] 📄 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have opened a PR on [truecharts/pub](https://github.com/truecharts/pub) adding the app icon.
- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md [hunk 1]
```diff
@@ -4,8 +4,47 @@
 
 This guide makes use of our command-line tool, called `TrueTool`.
 
-This should be installed by using:
-`pip install truetool`
+### How to Install
+Skip these steps if you have already installed TrueTool
+### Choose a folder
```
Comment (stavros-k) [inline docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:9]:
```suggestion
### How to Install

Skip these steps if you have already installed TrueTool

### Choose a folder
```

Comment (stavros-k) [reply docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:9]: markdown wants empty lines after and before headings

**Review by stavros-k — CHANGES_REQUESTED** [review]

File: docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md [hunk 2]
```diff
@@ -4,8 +4,47 @@
...
+
+It's important to save the script in a folder that is persistent across TrueNAS System Updates.
+This saves you from reinstalling or experiencing an accidental lack-of-backups after an update.
+
+##### New dataset
+
+In this example we created a `scripts` dataset on the TrueNAS SCALE system, feel free to use another folder.
+
+##### Root folder
```
Comment (stavros-k) [inline docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:18]:
```suggestion
#### New dataset

In this example we created a `scripts` dataset on the TrueNAS SCALE system, feel free to use another folder.

#### Root folder
```

Comment (stavros-k) [reply docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:18]: markdown does not like increasing heading more than +1 of the previous :)

File: docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md [hunk 3]
```diff
@@ -4,8 +4,47 @@
...
+The `/root` folder houses files for the root user.
+It's also persistent across updates and hence can be safely used for storing the script.
+
+### Open a Terminal
+
+**Change Directory to your scripts folder**
+
+```
+cd /mnt/pool/scripts
+```
```
Comment (stavros-k) [inline docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:29]:
```suggestion
```bash
cd /mnt/pool/scripts
```
```

File: docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md [hunk 4]
```diff
@@ -4,8 +4,47 @@
...
+
+**Git Clone truetool**
+
+```
+git clone https://github.com/truecharts/truetool.git
+```
```
Comment (stavros-k) [inline docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:35]:
```suggestion
```bash
git clone https://github.com/truecharts/truetool.git
```
```

File: docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md [hunk 5]
```diff
@@ -4,8 +4,47 @@
...
+
+**Change Directory to truetool folder**
+
+```
+cd truetool
+```
```
Comment (stavros-k) [inline docs/manual/SCALE Apps/Quick-Start Guides/14-backup-restore.md:41]:
```suggestion
```bash
cd truetool
```
```

**Review by Ornias1993 — CHANGES_REQUESTED** [review]
I actually don't want to maintain this in two places
A forward to the TrueTool github (readme) should be enough...

Untill we figure out some automation here or get rid of the backup guidance

**Pull request closed without merging**